src/images/falseColourPanel.component.tsx

```diff
@@ -99,127 +208,175 @@ const FalseColourPanel = (props: FalseColourPanelProps) => {
-            id="colour-map-select"
-            value={selectColourMap}
-            label="Colour Map"
-            onChange={handleColourMapChange}
-          >
-            <MenuItem value="">
-              <em>Default</em>
-            </MenuItem>
-            {colourMaps?.map((colourMap) => (
-              <MenuItem key={colourMap} value={colourMap}>
-                {colourMap}
-              </MenuItem>
-            ))}
-          </Select>
+          {colourMaps &&
+            extendedColourMap &&
+            ColourMapSelectExtended(
+              selectColourMap,
+              handleColourMapChange,
+              filteredColourMaps
+            )}
+          {colourMaps &&
+            !extendedColourMap &&
+            ColourMapSelectMain(
+              selectColourMap,
+              handleColourMapChange,
+              filteredColourMapsMain
+            )}
         </FormControl>
-        {/* TODO: is it better UX for these to be separate sliders or one range slider? */}
         <FormControl disabled={!enabled}>
-          <FormLabel id="lower-level-label" sx={{ margin: 'auto' }}>
-            Lower Level (LL)
+          <FormLabel id="range-slider-label" sx={{ margin: 'auto' }}>
+            Level Range
           </FormLabel>
           <Slider
             disabled={!enabled}
-            aria-labelledby="lower-level-label"
-            value={sliderLowerLevel}
+            aria-labelledby="range-slider-label"
+            value={[sliderLowerLevel, sliderUpperLevel]}
             valueLabelDisplay="auto"
             marks={marks}
             track={false}
-            onChange={(event, value) => {
-              if (typeof value === 'number') {
-                if (value > sliderUpperLevel) {
-                  // have to set it to the min as onChange isn't called for every number
-                  setSliderLowerLevel(sliderUpperLevel);
-                } else {
-                  setSliderLowerLevel(value);
-                }
+            onChange={(event, newValue) => {
+              if (Array.isArray(newValue)) {
+                const [lower, upper] = newValue;
+                setSliderLowerLevel(lower);
+                setSliderUpperLevel(upper);
               }
             }}
-            onChangeCommitted={(event, value) => {
-              if (typeof value === 'number') {
-                if (value > sliderUpperLevel) {
-                  // have to set it to the min as onChange isn't called for every number
-                  changeLowerLevel(sliderUpperLevel);
-                } else {
-                  changeLowerLevel(value);
-                }
+            onChangeCommitted={(event, newValue) => {
+              if (Array.isArray(newValue)) {
+                const [lower, upper] = newValue;
+                changeLowerLevel(lower);
+                changeUpperLevel(upper);
               }
             }}
             min={0}
             max={255}
             sx={{
```

Comment:
We can get rid of all this styling if we remove the `track={false}` bit - as MUI already has the indicators of the selected vs non-selected range. This greying out stuff was needed for the separate sliders to indicate that the min slider must be below the max slider and vice versa, but with them both on the same track this is no longer necessary and we can use the in-built MUI styling